README.md

```diff
@@ -1,6 +1,6 @@
 # PSDesiredStateConfiguration module
 
-**NOTE: We are currently NOT accepting PRs for this project**
+**NOTE: A new version of this platform is in development. Although the direction is language agnostic, the vision includes compatability with existing commmunity resources. Please submit your feedback on [DSC version 3](https://github.com/powershell/dsc).**
```

Comment:
```suggestion
**NOTE: The next generation of this platform is in development. Please submit your feedback on [DSC v3](https://github.com/powershell/dsc).**
```

Reply:
We can use the built-in alert syntax:
```suggestion
> [!NOTE]
> The next generation of this platform is in development. Please submit your feedback on [DSC v3](https://github.com/powershell/dsc).
```
Which renders as:

> [!NOTE]
> The next generation of this platform is in development. Please submit your feedback on [DSC v3](https://github.com/powershell/dsc).